miniapp/USERGUIDE.md

```diff
@@ -381,6 +381,17 @@ class CustomAdDisplayer: MiniAppAdDisplayer {
 miniAppMessageBridge.setAdMobDisplayer(CustomAdDisplayer())
 ```
 
+### #6 Screen Orientation
+The default setting does not allow miniapp to change hostapp screen orientation.
+Hostapp can allow miniapp to control the screen orientation for better experience by calling 
+```kotlin
+miniAppMessageBridge.allowScreenOrientation(true)
+```
+
+In case miniapp is allowed to control, please ensure that your activity handles screen orientation.
+There are several ways to prevent the view from being reset.
+In our Demo App, we set the config on activity. See [here](https://developer.android.com/guide/topics/resources/runtime-changes#HandlingTheChange).
```

Comment:
Can we also give a basic example? For example that they can add `android:configChanges="orientation|screenSize"` to their manifest?